vars/pythonSetupPyPipeline.groovy

```diff
@@ -299,6 +299,9 @@ def initParameterWithBaseValues(Map pipelineParams) {
   pipelineParams["dockerRunArgs"] = pipelineParams.dockerRunArgs ?: ""
   pipelineParams["dockerRunArgs"] += " -v /etc/passwd:/etc/passwd:ro -v /opt/jenkins/.ssh:/opt/jenkins/.ssh:ro --network host"
   pipelineParams["pypiRepo"] = pipelineParams.pypiRepo ?: "https://test.pypi.org/legacy/"
+  pipelineParams["buildCommand"] = pipelineParams.buildCommand ?: "python setup.py build"
```

Comment:
can I disable one of the steps, lint for example?

Reply:
Unfortunately no, this PR is only to make commands customizable, not steps setup itself

Reply:
In this case, if you want to skip it,  the workaround is to make a 'dummy command' inside

Reply:
Worth to create an issue for making steps optional. @askalski85 can you please create a feature request for this. Thanks

Reply:
it looks good we could just document in readme that dummy command would be sufficient